internal/registry/ui/registry.css

```
@@ -0,0 +1,412 @@
+:root {
+  --ink: #102a35;
+  --deep-water: #173f4d;
+  --lagoon: #167f78;
+  --sea-glass: #9bd5c6;
+  --mist: #e8f2ee;
+  --paper: #f7faf7;
+  --signal: #e66f45;
+  --quiet: #5e7377;
+  --line: #bfd2cc;
+  --shadow: 0 24px 60px rgb(16 42 53 / 10%);
+  font-family: "Avenir Next", Avenir, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)